Documentation/guides/messages/README.md

```diff
@@ -55,17 +55,14 @@ ms.date: 01/24/2020
 + [XA0107](xa0107.md): `{Assmebly}` is a Reference Assembly.
 + [XA0108](xa0108.md): Could not get version from `lint`.
 + [XA0109](xa0109.md): Unsupported or invalid `$(TargetFrameworkVersion)` value of 'v4.5'.
-+ [XA0110](xa0110.md): Disabling $(AndroidExplicitCrunch) as it is not supported by `aapt2`. If you wish to use $(AndroidExplicitCrunch) please set $(AndroidUseAapt2) to false.
```

Comment:
As with various musings elsewhere, I don't think we want to *remove* these codes from the list.  We instead want to say that they're obsolete or something.